src/DynamoRevit/DynamoRevitApp.cs

```diff
@@ -110,6 +110,39 @@ private static string GetDynamoRoot(string dynamoRevitRoot)
         private static EventHandlerProxy proxy;
         private AddInCommandBinding dynamoCommand;
 
+        private Result loadDependentComponents()
+        {
+            var dynamoRevitRootDirectory = Path.GetDirectoryName(Path.GetDirectoryName(assemblyName));
+            var versionNumber = ControlledApplication.VersionNumber;
+
+            var dynamoRevitAditionsPath = Path.Combine(dynamoRevitRootDirectory,
```

Comment:
assemblyName is full path for DynamoRevit.dll, you could have used that directly.